Pod/Classes/NotificationConvenience.swift

```diff
@@ -19,7 +19,7 @@ public extension NotificationCenter {
     
     // Needed for objc compatibility
     public func post(notification: Notification) -> Void {
-        post(notification)
+        post(notification:notification, queue: DispatchQueue.main)
```

Comment:
ah yeah good catch.... I'm still in favor of going against the swifty way and going back to prefixing with `urbn_` 🙈